### PR TITLE
PDCT 187 add download all results

### PR DESCRIPTION
**Pull request by PatFawbertMills**
- Add link to download database
- Update e2e tests to include both links

### CODE REVIEW

File: src/pages/search/index.tsx [hunk 1]
```diff
@@ -287,18 +288,27 @@ const Search = () => {
...
                       href="#"
                       className="text-blue-600 hover:underline flex gap-2 items-center justify-end"
                       data-cy="download-search-csv"
                       onClick={handleDownloadCsvClick}
                     >
-                      {downloadCSVStatus === "loading" && <Loading />} Download search results (.csv)
+                      {downloadCSVStatus === "loading" && <Loading />} this search
                     </a>
+                    <span>|</span>
+                    <ExternalLink
```
Comment (diversemix) [inline src/pages/search/index.tsx:302]: Won't this also show while loading?

**Review by diversemix — COMMENTED** [review]
Can't see the e2e tests updated :shrug: .... did you miss the file(s) on commit?

File: src/pages/search/index.tsx (hunk 1 above)
Comment (PatFawbertMills) [reply src/pages/search/index.tsx:302]: Yes that's OK as both options are always available to click on, when loading the loading indicator is displayed next to the link that says "this search"

**Comment by PatFawbertMills** [comment]
It is in the file `e2e/cypress/e2e/cclw/pages/search.cy.js`

File: src/pages/search/index.tsx (hunk 1 above)
Comment (diversemix) [reply src/pages/search/index.tsx:302]: okies -sgtm

**Review by diversemix — APPROVED** [review]
Wonderful - sorry about being slow today :grin:

**Pull request merged**